.expeditor/scripts/bk_container_prep.sh

```diff
@@ -18,6 +18,8 @@ echo "--- Preparing Container..."
 
 export FORCE_FFI_YAJL="ext"
 export CHEF_LICENSE="accept-no-persist"
+export ENV["CHEF_LICENSE_SERVER"] = "http://hosted-license-service-lb-8000-606952349.us-west-2.elb.amazonaws.com:8000/"
```

Comment:
Feels like these env vars should be set in Buildkite itself.

Reply:
will refactor them to be in a single place instead of spread out throughout the codebase.